onyo/lib/onyo.py

```diff
@@ -478,11 +478,24 @@ def is_inventory_path(self,
             Path to check.
         """
 
-        return path.is_relative_to(self.git.root) and \
+        return self.git.root in path.parents and \
             not self.git.is_git_path(path) and \
             not self.is_onyo_path(path) and \
             not self.is_onyo_ignored(path)
 
+    def is_item_path(self, path: Path) -> bool:
+        r"""Whether ``path`` is a valid path for an item.
+        """
+        return path == self.git.root or self.is_inventory_path(path) or self.is_template_path(path)
+
+    def is_template_path(self, path) -> bool:
+        r"""Whether ``path`` is a valid template location.
```

Comment:
`ruff` dislikes this. When there is no description, the `"""` should be on the same line as the subject.